### DISCUSSION TITLE
Change the color debug log

### DISCUSSION THREAD

**Discussion by nicnocquee** [coollabsio/coolify#2482] [Feature Requests]
Currently it's red but usually red color indicates errors. This make users (well, me) think there are lots of errors during the build.

![image](https://github.com/coollabsio/coolify/assets/311343/83334cfb-176b-4058-b804-f77020f265c2)

**Comment by andrasbacsai** [comment]
Oh that should not be red! Making in an issue about this.

**Comment by peaklabs-dev** [comment]
This is now available.
